sdk/python/setup.py

```diff
@@ -66,6 +66,7 @@
     "uvicorn[standard]>=0.14.0",
     "proto-plus<1.19.7",
     "tensorflow-metadata>=1.0.0,<2.0.0",
+    "dask>=2021.9.*",
```

Comment:
Should we perhaps relax this dependency more? Aren't we excluding a lot of dask versions by only having 2021.9+?

Reply:
fixed